ci/autoscaler/pipeline.yml

```diff
@@ -571,17 +571,18 @@ jobs:
   - put: mtar-bucket
     params:
       file: "build/artifacts/app-autoscaler-release-v*.mtar"
+  - put: app-autoscaler-release
+    params:
+      rebase: true
+      repository: app-autoscaler-release
+      tag: build/tag # ⚠️ Remove this again, if it breaks the subsequent step.
```

Comment:
The [Readme](<https://github.com/concourse/github-release-resource/blob/b139c8292824c6da0761e75d51351031c78b2711/README.md>) for that concourse-ressource does not specify, if it fails when a tag is already in use. This probably just depends on the behaviour of <https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release> which as well is not specified in this case.

Setting it here might be beneficial anyway in the (although unlikely) case that a PR gets merged after running this step and before the next one.